dts/bindings/fuel-gauge/battery_cutoff.yaml

```
@@ -0,0 +1,28 @@
+#
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+
+  Properties for fuel-gauges that may control battery cutoff, this is common in SBS compliant or
+  similarly smart battery fuel gauges.
+
+  Note: These properties are to be used with meaningful defaults in fuel gauge ICs that implement
+  battery cutoff support. See the default fuel gauge SBS Gauge compatible as an example.
+
+properties:
+  battery_cutoff_support:
```

Devicetree [property names](https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#naming-conventions) should not use underscores.
```suggestion
  battery-cutoff-support:
```